abis_mapping/templates/dwc_mvp/schema.json

```diff
@@ -37,17 +48,87 @@
         "maximum": 180.0
       }
     },
+    {
+      "name": "geodeticDatum",
+      "title": "Geodetic Datum",
+      "description": "...",
+      "example": "WGS84",
+      "type": "string",
+      "format": "default",
+      "constraints": {
+        "required": true,
+        "enum": [
+          "AGD84",
+          "EPSG:4203",
+          "GDA2020",
+          "EPSG:7844",
+          "GDA94",
+          "EPSG:4283",
+          "WGS84",
+          "EPSG:4326"
+        ]
+      }
+    },
+    {
+      "name": "coordinateUncertaintyInMeters",
+      "title": "Coordinate Uncertainty In Meters",
+      "description": "...",
+      "example": "50.0",
+      "type": "number",
+      "format": "default",
+      "constraints": {
+        "required": false,
+        "minimum": 0.0
+      }
+    },
+    {
+      "name": "dataGeneralizations",
+      "title": "Data Generalizations",
+      "description": "...",
+      "example": "Coordinates rounded to the nearest 10 km for conservation concern",
+      "type": "string",
+      "format": "default",
+      "constraints": {
+        "required": false
+      }
+    },
     {
       "name": "eventDate",
       "title": "Event Date",
       "description": "...",
-      "example": "2019-09-23",
+      "example": "2019-09-23T14:03+08:00",
       "type": "timestamp",
       "format": "default",
       "constraints": {
         "required": true
       }
     },
+    {
+      "name": "samplingProtocol",
+      "title": "Sampling Protocol",
+      "description": "...",
+      "example": "human observation",
+      "type": "string",
+      "format": "default",
+      "constraints": {
+        "required": false,
+        "enum": [
+          "human observation",
```

Comment:
This is a very short list of values for this enum but probably not much you can do about it

Reply:
@kehh I agree, I don't think I've been given the full lists yet though